src/engraving/engravingproject.cpp

```diff
@@ -95,7 +95,7 @@ Err EngravingProject::setupMasterScore()
 Err EngravingProject::doSetupMasterScore(Ms::MasterScore* score)
 {
     TRACEFUNC;
-
+    score->createPaddingTable();
```

Comment:
please, return an empty line

Reply:
done!

Reply:
I meant this line

<img width="1368" alt="image" src="https://user-images.githubusercontent.com/10116828/165905327-8a36fd96-928b-4454-979d-e70b89795749.png">

Reply:
ah sorry! I misunderstood. Should be good now